generators/pythonv2/codegen/src/ast/Variable.ts

```diff
@@ -0,0 +1,37 @@
+import { AstNode } from "./core/AstNode";
+import { Writer } from "./core/Writer";
+import { Annotation } from "./Annotation";
+
+export declare namespace Variable {
+    interface Args {
+        /* The name of the variable */
+        name: string;
+        /* The type annotation of the variable */
+        type?: Annotation;
+        /* The initializer for the variable */
+        initializer?: string;
+    }
+}
+
+export class Variable extends AstNode {
+    public readonly name: string;
+    public readonly type: Annotation | undefined;
```

Comment:
Once/if this becomes `Field` then type should not be nullable

Reply:
annotations are optional in python though? eg this could be `color = 'red'`

Reply:
They're not really optional for fields, which is what this class is supposed to be (not variables)